pkgs/development/libraries/libcouchbase/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, cmake, pkgconfig, libevent, openssl}:
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, libevent, openssl, fetchpatch }:
```

Comment:
```suggestion
{ stdenv, fetchFromGitHub, cmake, pkgconfig, libevent, openssl }:
```

Reply:
Good catch! Forgot to remove this after I realized that the patch can't be applied to 2.10.4 as-is :smile:

Reply:
I probably end up doing this even a couple times a week also :smile: